### DISCUSSION TITLE
Shinydashboard tidak bisa dideploy

### DISCUSSION THREAD

**Discussion by nikenparwati** [teamalgoritma/community#925] [Q&A] [answered]
Dear teamalgoritma,
Saya mengalami masalah dengan deploy shinydashboard saya.  
Saya sdh daftar seminggu lalu. Tapi sekarang shinyapps nya tidak bisa dibuka juga. Terima kasih

**Comment by ranydwi** [comment]
Dear Bu @nikenparwati ,

Pertanyaan Ibu sudah kami terima dan akan dibalas oleh mentor kami. Namun, sebelum itu bolehkah ditambahkan penjelasan bagian shinyapps yang tidak bisa dibuka dibagian mana ya?

Regards,
Team Algoritma

**Comment by nikenparwati** [comment, marked as answer]
Halo mbak Rany. Alhamdulillah sdh bisa mbak, laptops saya restart. Terima
kasih

Pada Rab, 16 Okt 2024 pukul 15.58 Rany Dwi Cahyaningtyas <
***@***.***> menulis:
